src/potentials.cpp

```diff
@@ -731,18 +731,18 @@ void HardSphere::initPairMatrices() {
 // =============== Hertz ===============
 
 void Hertz::initPairMatrices() {
-    const TExtractorFunc extract_radius = [](const AtomData &a) -> double { return a.hdr; };
+    const TExtractorFunc extract_diameter = [](const AtomData &a) -> double { return 2*a.hdr; };
     const TExtractorFunc extract_epsilon = [](const AtomData &a) -> double { return a.eps_hertz; };
-    const TCombinatorFunc comb_radius = PairMixer::getCombinator(combination_rule, PairMixer::COEF_SIGMA);
+    const TCombinatorFunc comb_diameter = PairMixer::getCombinator(combination_rule, PairMixer::COEF_SIGMA);
     const TCombinatorFunc comb_epsilon = PairMixer::getCombinator(combination_rule, PairMixer::COEF_EPSILON);
 
-    hydrodynamic_diameter = PairMixer(extract_radius, comb_radius, [](double x) -> double { return 2*x; })
+    diameter_squared = PairMixer(extract_diameter, comb_diameter, &PairMixer::modSquared)
```

Comment:
`sigma_squared` like in LJ, WCA, hardsphere?

Reply:
I would keep the naming scheme as it is so far. However I already have a plan how to refactorize `AtomData` and pair potentials together with names changes.